dataline-integrations/readme.md

```diff
@@ -0,0 +1,12 @@
+### Upgrading an Integration
+1. Make sure you can build the integration locally
+
+    ```
+    ./tools/integrations/manage.sh build <path to integration>
```

Comment:
Should we remove the gradle `buildImage` tasks then?

Reply:
nope, manage uses it

Reply:
nice